ioservice/io_fops.c

```diff
@@ -678,13 +673,9 @@ M0_INTERNAL int m0_ioservice_fop_init(void)
    The IO coalescing subsystem from ioservice primarily works on IO segments.
```

Comment:
I guess this part could be removed too. We don't have "IO coalescing subsystem that works on IO segments" anymore.